src/python/pants/backend/python/goals/run_pex_binary.py

```diff
@@ -114,12 +123,62 @@ def in_chroot(relpath: str) -> str:
     ]
     extra_env = {
         **complete_pex_env.environment_dict(python_configured=pex.python is not None),
-        "PEX_PATH": in_chroot(local_dists.pex.name),
+        "PEX_PATH": _in_chroot(local_dists.pex.name),
         "PEX_EXTRA_SYS_PATH": os.pathsep.join(source_roots),
     }
 
     return RunRequest(digest=merged_digest, args=args, extra_env=extra_env)
 
 
+@rule
+async def run_pex_debug_adapter_binary(
+    field_set: PexBinaryFieldSet,
+    debugpy: DebugPy,
+    debug_adapter: DebugAdapterSubsystem,
+) -> RunDebugAdapterRequest:
+    if field_set.run_in_sandbox:
+        logger.warning(
+            softwrap(
+                f"""
+                Using `run --debug-adapter` on the target {field_set.address}, which sets the field
+                `run_in_sandbox` set to `True`. This will likely cause your breakpoints to not be hit,
```

Comment:
You still say "set" twice. 

Sorry, I normally wouldn't be this picky if it were a comment. But because it's documentation, I think it matters more.

Reply:
No need to apologize, I wanna get this right as well. Sorry my self-review-goggles are fuzzy today,